Makefile

```diff
@@ -86,6 +86,12 @@ docker-run:
 		-e SLAPD_CONFIG_ROOTPW="secret" \
 		-p "389:389" -p "636:636" \
 		-d cobaugh/openldap-alpine
+	docker run \--name cratedb \
```

Comment:
Do we need to remove the backslash here? `\--name`

Reply:
3b22f2eefaf9553dbc653fce135893f37d213707